XKCD_colour_definition.py

```diff
@@ -0,0 +1,950 @@
+XKCD_RGB_TO_COLOR_NAMES = {(0.6745098039215685, 0.76078431372549, 0.8509803921568628): ['XKCD_cloudy_blue'],
```

Comment:
You could totally use these values, but it seems that original content from xkcd defines also RGB 3-uple integer, or at least standard hexa counterparts (I'm refering to https://xkcd.com/color/rgb.txt ). It seems to be a much more solid definition than a float approximation.

Reply:
Let's add also that we don't want to mix 'web' format with 'xkcd' one, and the current structure of `colour` is quite friendly to add new formats. So why not use a similar table without the `XKCD_` prefix. We would then wait for an attribute `Colour().xkcd` that could be read or written to, and a instanciation time keyword `Colour(xkcd='light eggplant')`. You could actually start the `xkcd` realm as an exact copy of `web` realm, but using simply another correspondance table... This would also give you the defaulting of using hexa decimal forms when no xkcd definition exists.